### PR TITLE
Backport suppress dispatching to PDF viewer if plugins are disabled to 1.6.x

### PR DESCRIPTION
**Pull request by rreimann**
This PR backports [#9507](https://github.com/electron/electron/pull/9507#issuecomment-303226315) to the 1.6.x branch.

🍒

### CODE REVIEW

File: atom/browser/web_contents_preferences.cc [hunk 1]
```diff
@@ -207,9 +209,23 @@ bool WebContentsPreferences::IsSandboxed(content::WebContents* web_contents) {
...
+  bool bool_value = false;
+  web_preferences.GetBoolean(attribute_name, &bool_value);
+  return bool_value;
+}
+
+bool WebContentsPreferences::IsSandboxed(content::WebContents* web_contents) {
+  return IsPreferenceEnabled("sandbox", web_contents);
+}
+
+bool WebContentsPreferences::UsesNativeWindowOpen(
```
Comment (kevinsawicki) [inline atom/browser/web_contents_preferences.cc]: `nativeWindowOpen` was added in 1.7.1 so this method can be removed here.

File: atom/browser/web_contents_preferences.h [hunk 2]
```diff
@@ -37,7 +37,11 @@ class WebContentsPreferences
   static void AppendExtraCommandLineSwitches(
       content::WebContents* web_contents, base::CommandLine* command_line);
 
+  static bool IsPreferenceEnabled(const std::string& attribute_name,
+                                  content::WebContents* web_contents);
   static bool IsSandboxed(content::WebContents* web_contents);
+  static bool UsesNativeWindowOpen(content::WebContents* web_contents);
```
Comment (kevinsawicki) [inline atom/browser/web_contents_preferences.h]: `nativeWindowOpen` was added in 1.7.1 so this method can be removed here.

File: atom/browser/web_contents_preferences.cc (hunk 1 above)
Comment (rreimann) [inline atom/browser/web_contents_preferences.cc]: Sorry, didn't look close enough at the diff. Should be removed now.

**Comment by kevinsawicki** [comment]
Thanks @rreimann

**Pull request merged**